bot/exts/info/information.py

```diff
@@ -256,6 +256,9 @@ async def create_user_embed(self, ctx: Context, user: MemberOrUser) -> Embed:
             name = f"{user.nick} ({name})"
         name = escape_markdown(name)
 
+        if passed_as_message:
+            name += " - From Message"
```

Comment:
```suggestion
            name += " (From Message ID)"
```

Reply:
I decided to use a hyphen instead of brackets since it looked weird when the user has a nickname

Reply:
Ah, that makes sense.